src/AbstractEmail.php

```diff
@@ -0,0 +1,6 @@
+<?php
+namespace Pokedex;
+
+abstract class AbstractEmail
```

Comment:
its common to put abstracts into an abstract folder

Reply:
Created in ed07096